SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialGameInstance.cpp

```diff
@@ -67,7 +67,8 @@ bool USpatialGameInstance::HasSpatialNetDriver() const
 
 void USpatialGameInstance::CreateNewSpatialWorkerConnection()
 {
-	SpatialConnection = NewObject<USpatialWorkerConnection>();
+	SpatialConnection = NewObject<USpatialWorkerConnection>(this);
+	SpatialConnection->Init(GetWorld());
```

Comment:
Isn't this going to be null at the point of initilisation for StartPlayInEditorGameInstance and StartGameInstance?

Reply:
You mean `World` will be?

Reply:
From what I've seen, we always have a `World` at this point